src/chromecast.js

```diff
@@ -168,6 +169,30 @@ export default class ChromecastPlugin extends UICorePlugin {
     }
   }
 
+  enableCaptions(enabled) {
+    if (!this.session) return;
+    var enabledTextTrackIDs = [];
+    if (enabled && this.textTracks && this.textTracks.length > 0) {
+      enabledTextTrackIDs = [this.textTracks[0].id];
+    }
+    this.session.sendMessage(
+      'urn:x-cast:boxcast:active-text-tracks',
```

Comment:
Would be nice to pull this namespace through the options, along with our appId.

Reply:
I mean, it literally only works with the BoxCast custom receiver app(s)... that's why I said this would/could never be merged to mainline. What's the benefit to passing it in?

Reply:
My only thinking was that if we wanted to provide docs on how to use this with a custom receiver app perhaps it _could_ go back into mainline.

Reply:
I added an option to customize the message namespace.